mismi-ec2/src/Mismi/EC2/Config/Virtualization.hs

```diff
@@ -0,0 +1,120 @@
+{-# LANGUAGE NoImplicitPrelude #-}
+{-# LANGUAGE OverloadedStrings #-}
+module Mismi.EC2.Config.Virtualization (
+    A.InstanceType
+  , virtualizationFor
+  ) where
+
+import qualified Mismi.EC2.Amazonka as A
+
+import           Mismi.EC2.Data
+
+virtualizationFor :: A.InstanceType -> Virtualization
```

Comment:
My bad for the code in the first place, but given it is a bit more prominent here I think there needs to be a comment explaining what this is. Specifically it is providing a "preferred" virtualization for each instance type. Where "preferred" means: if it supports HVM prefer HVM, otherwise prefer PV.